unit-tests-java8/src/test/java/org/eclipse/collections/test/lazy/DropWhileIterableTestNoIteratorTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2016 Goldman Sachs.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.test.lazy;
+
+import org.eclipse.collections.api.LazyIterable;
+import org.eclipse.collections.api.list.MutableList;
+import org.eclipse.collections.impl.block.factory.Predicates;
+import org.eclipse.collections.impl.lazy.DropWhileIterable;
+import org.eclipse.collections.impl.test.junit.Java8Runner;
+import org.eclipse.collections.test.IterableTestCase;
+import org.eclipse.collections.test.LazyNoIteratorTestCase;
+import org.eclipse.collections.test.list.mutable.FastListNoIterator;
+import org.junit.runner.RunWith;
+
+@RunWith(Java8Runner.class)
+public class DropWhileIterableTestNoIteratorTest implements LazyNoIteratorTestCase
```

Comment:
This follows the same convention as `DropIterableTestNoIteratorTest`. 
I will rename it to `DropIterableNoIteratorTest`.